### DISCUSSION TITLE
[Question] Any convention to API-document a namespace?

### DISCUSSION THREAD

**Discussion by acolomitchi** [dotnet/docfx#8271] [General]
Is it possible to create/include an API documentation for a namespace?
If positive, how?

**Comment by KalleOlaviNiemitalo** [comment]
I didn't try, but this should be possible using an [Overwrite File](<https://dotnet.github.io/docfx/tutorial/intro_overwrite_files.html>) with the correct `uid` and property name (e.g. `remarks`).

**Comment by KalleOlaviNiemitalo** [comment]
See <https://github.com/dotnet/docfx/issues/952#issuecomment-264776462> for a sample overwrite file.
